layersvt/device_simulation.cpp

```diff
@@ -73,16 +75,54 @@ const uint32_t kLayerPropertiesCount = (sizeof(kLayerProperties) / sizeof(kLayer
 const VkExtensionProperties *kExtensionProperties = nullptr;
 const uint32_t kExtensionPropertiesCount = 0;
 
+#if !defined(__ANDROID__)
```

Comment:
Yes, it is just to avoid an unused variable to get rid of the "unused variable" error reported by the ndk-build.